static/app/components/profiling/flamegraph/flamegraphSpans.tsx

```diff
@@ -127,8 +127,8 @@ export function FlamegraphSpans({
 
     const drawSpans = () => {
       spansRenderer.draw(
-        spansView.configView.transformRect(spansView.configSpaceTransform),
```

Comment:
This wasnt taking into account the offset config space and was rendering the span bars slightly differently